play/PlayerRules/CharacterCreation/03_Feats.txt

```diff
@@ -466,62 +452,101 @@ Proficient: Explosives
+not run out until after this turn, its effect wraps around to the beginning of 
+your next turn. Cannot be used in concert with Run 'N' Gun.
+
+Cover Slide [15 Stamina]:
+Prerequisites: DEX 4
 Move up to your speed and end next to a barrier behind which you can take 
-cover. You hunker behind that cover as a free action. Cannot be used in 
-concert with run 'n' gun.
+cover. You hunker or go prone behind that cover as a free action. Cannot 
+be used in concert with Run 'N' Gun.
+
+Empty the Mag: 
+Prerequisites:
+It happens. You aren't quite sure if you can hit your target perhaps, or 
+maybe you round that corner and are staring down the barrel of an enemy AR-68.
+Whatever the case, you need to dump the mag in their face to guarantee your 
+safety. Roll to pass a DC 30% DEX check. on success, you may fire off the 
+remainder of your clip or magazine (Applies only to weapons with a semi-automatic
+mode, maximum 10 shots). Roll hits for each shot. Using this feat takes 
+3 actions, and must be completed in the span of one turn. After using 
+this feat, the user must reload their weapon for it to be effective again. 
+Can only be used once per encounter, LVL number of times per day. At level 5 
+and beyond, can be used twice per encounter.
 
-Flash Dance: 
+Flash Dance [8*X Stamina]: 
 Prerequisites:
-Expend 10*lvl Nanites. Allows the user to make Lvl number of attacks with 
-their melee weapon within a 5m space, rolling for each strike. All bonuses 
-that would normally apply to a standard attack still apply.
+For 1 Action, the user can make up to Lvl/2+1 (round up) number of attacks 
+with their melee weapon within a 5m space, rolling for each strike, where 
+X is the number of attacks. All bonuses that would normally apply to a standard
+attack still apply.
 
-Gun Kata [30 Nanites / Turn]: 
+Gun Kata [20 Stamina]: 
 Prerequisites: Level 10 or higher
-For up to 3rds, reduce weapon range to 15m, reduce Miss Chance by 50%. Treat 
-any weapon as if it were a CQB weapon. After a successful strike, you may roll 
-for an additional strike during the same action on any target within the 15m 
-range.
+For 1 Action, reduce weapon range to 15m and reduce Miss Chance by 5. Treat 
+any weapon as if it were a CQB weapon. After a successful hit, you may roll 
+for an additional hit on any target within the 15m range, for up to 3 rounds
+fired. If you miss a shot, your Gun Kata ends.
+
+Kata [Minimum 5 Stamina]: 
+Prerequisites: Level 10 or higher
+Whenever you land a successful melee hit against someone, you may roll for X 
+additional attacks, where X is your stamina spent/5. Each additional attack has an additional
+2 miss chance. (A 10th level character making their 3rd strike in a row would be 
+rolling at +4 miss chance). This +2 debuff applies to a Monk's Shadow Strike as well,
+increasing the DC by 20 per strike instead of increasing the strike's miss chance.
 
-Multi-Target [ X*40 Nanites]: 
+Multi-Target [13*X Stamina]: 

```

Comment:
Chance to 4 / 2 actions?

Reply:
@trowl223  I think it would be too weak compared to just a gun-kata. Also, i just realized we didn't add a clause for what 'roll' people make to see if you succeed with multi-target. So I did roll to hit 'your first target'? Which means players will always choose the easiest one..